lib/fluent/supervisor.rb

```diff
@@ -437,28 +437,10 @@ def self.load_config(path, params = {})
       end
 
       log_level = params['log_level']
-      suppress_repeated_stacktrace = params['suppress_repeated_stacktrace']
-      ignore_repeated_log_interval = params['ignore_repeated_log_interval']
-      ignore_same_log_interval = params['ignore_same_log_interval']
 
-      log_path = params['log_path']
       chuser = params['chuser']
       chgroup = params['chgroup']
       chumask = params['chumask']
```

Comment:
These remaining local variables doesn't seem need too, passing them directly to `se_config` will make the code makes simple.

Reply:
Indeed! I will fix it!

Reply:
Fixed!